nav2_smac_planner/include/nav2_smac_planner/constants.hpp

```diff
@@ -59,6 +67,38 @@ inline MotionModel fromString(const std::string & n)
   }
 }
 
+inline std::string toString(const GoalHeading & n)
+{
+  switch (n)
+  {
+    case GoalHeading::DEFAULT:
+      return "DEFAULT";
+    case GoalHeading::BIDIRECTIONAL:
+      return "BIDIRECTIONAL";
+    case GoalHeading::ANY_HEADING:
+      return "ANY_HEADING";
+    default:
+      return "Unknown";
+  }
+}
+
+// TODO(@stevedanomodolor) find a better way to do this
```

Comment:
What's the concern? I think this works fine, anything more you want to expand on here?